randovania/games/prime1/exporter/game_exporter.py

```diff
@@ -197,3 +212,15 @@ def export_game(self, patch_data: dict, export_params: GameExportParams,
                 raise
             else:
                 raise RuntimeError(f"randomprime panic: {e}") from e
+            
+
+        if all(v == 1.0 for v in [random_scale_min, random_scale_max,
```

Comment:
Avoid having an empty if block with the logic on else. You can easily invert the criteria to `if any(v != 1.0 ...`